runtime/karura/src/lib.rs

```diff
@@ -749,6 +749,9 @@ parameter_type_with_key! {
 				// use the ED of currency_id_0 as the ED of lp token.
 				if currency_id_0 == GetNativeCurrencyId::get() {
 					NativeTokenExistentialDeposit::get()
+				} else if let CurrencyId::Erc20(_) = currency_id_0 {
+					// LP token with erc20
+					1
```

Comment:
Maybe `Zero::zero()` is better.

Reply:
we may want to allow governance to set ed for lp token with erc20 involved.
@wangjj9219 what do you think?

Reply:
> we may want to allow governance to set ed for lp token with erc20 involved.
> @wangjj9219 what do you think?

agree, these assets will be added frequently.